rules/integrations/google_workspace/defense_evasion_application_removed_from_blocklist_in_google_workspace.toml

```diff
@@ -26,7 +26,51 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "Application Removed from Blocklist in Google Workspace"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Application Removed from Blocklist in Google Workspace
+
+Google Workspace Marketplace is an online store for free and paid web applications that work with Google Workspace services and with third party software. Listed applications are based on Google APIs or on Google Apps Script and created by both Google and third party developers.
+
+Marketplace applications require access to specific Google Workspace resources. Applications can be installed by individual users, if permissions grant availability or can be installed for an entire Google Workspace domain by administrators. Consent screens typically display what permissions and privileges the application requires during install. As a result, malicious Marketplace applications may require more permissions than necessary or have malicious intent.
+
+Google clearly states that they are not responsible for any product on the Marketplace that originates from a source other than Google.
+
+This rule identifies a previously blocked application being manually removed from the Marketplace blocklist by a Google Workspace account with administrative privileges.
+
+#### Possible investigation steps
+
+- Identify user account(s) associated by reviewing `user.name` or `user.email` in the alert
+- This rule relies on data from `google_workspace.admin` thus indicating the associated user has administrative privileges to the Marketplace
+- With access to Google Workspace admin console, visit Security > Investigation tool with filters for the user email and event is `Assign Role` or `Update Role` to determine if new cloud roles were recently updated
+- With the user account, review other potentially related events within the last 48 hours
```

Comment:
Can you elaborate more on what you mean by "with the user account"?

Reply:
@nastasha-solomon - Line 43 the analyst would identify the user involved, this is then used check their activity historically within the last 48 hours for additional anomalies.

Reply:
@terrancedejesus I responded [here](https://github.com/elastic/detection-rules/pull/2540/files#r1162104669).